services/tenant-ui/frontend/src/store/innkeeper/innkeeperTenantsStore.ts

```diff
@@ -171,6 +172,50 @@ export const useInnkeeperTenantsStore = defineStore('innkeeperTenants', () => {
     return approveResponse;
   }
 
+  async function refreshCheckInPassword(
+    id: string,
+    email: string,
+    payload: any = {}
+  ) {
+    console.log('> reservationStore.refreshCheckInPassword');
+    error.value = null;
+    loading.value = true;
+
+    // Don't keep this as state, make sure the password doesn't hang around in memory
+    let refreshResponse: ApproveResponse = {};
+    await acapyApi
+      .putHttp(API_PATH.INNKEEPER_RESERVATIONS_REFRESH_PASSWORD(id), payload)
+      .then((res) => {
+        refreshResponse = res.data;
```

Comment:
When there is one statement you could do this as a single line. This is fine though. I'm not sure what the project preference is. 

Personally I like doing one liners without curly braces. I think it's cleaner, but not important.